packages/admin-ui/ui/src/domain/inventory/locations/components/address-form/index.tsx

```diff
@@ -113,7 +115,9 @@ const AddressForm = ({
             control={control}
             name={path("country_code")}
             rules={{
-              required: addressFieldsRequired,
+              required: addressFieldsRequired
+                ? "This field is required"
```

Comment:
nit(non-blocking): More specific error message

```suggestion
                ? FormValidator.required("Country")
```